lib/DDG/Rewrite.pm

```diff
@@ -131,6 +136,7 @@ sub _build_nginx_conf {
 	my $wrap_jsonp_callback = $self->has_callback && $self->wrap_jsonp_callback;
 	my $wrap_string_callback = $self->has_callback && $self->wrap_string_callback;
 	my $uses_echo_module = $wrap_jsonp_callback || $wrap_string_callback;
+        my $callback = $self->callback;
```

Comment:
@zachthompson simplified this line here. I also think I've fixed the spacing 'irregularities', but they sometimes sneak by me.